### PR TITLE
updates counter responsiveness and functionality

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixes counter, all, question and clear functions
- fixes tablet and mobile view

screen shot 2016-01-14 at 17 11 37
screen shot 2016-01-14 at 17 06 57
screen shot 2016-01-14 at 17 06 49

### CODE REVIEW

File: filer/private/sass/settings/_custom.scss [hunk 1]
```diff
@@ -44,7 +44,7 @@ $padding-large: 20px;
 
 $screen-mobile: 320px;
 $screen-tablet: 720px;
-$screen-desktop: 1024px;
+$screen-desktop: 975px;
```
Comment (Brankub) [inline filer/private/sass/settings/_custom.scss]: maybe it's better to create new one breakpoint? Or it will fix all other places too?

Comment (lory87) [inline filer/private/sass/settings/_custom.scss]: @Brankub I didn't create a new because this is nowhere else used

Comment (Brankub) [inline filer/private/sass/settings/_custom.scss]: :+1:

**Comment by Brankub** [comment]
:+1:

**Pull request merged**